CredentialProvider.Microsoft/CredentialProviders/Vsts/IAuthUtil.cs

```diff
@@ -17,16 +17,24 @@ public interface IAuthUtil
     {
         Task<Uri> GetAadAuthorityUriAsync(Uri uri, CancellationToken cancellationToken);
 
-        Task<bool> IsVstsUriAsync(Uri uri);
+        Task<IFeedUriSource> GetFeedUriSource(Uri uri);
 
         Task<Uri> GetAuthorizationEndpoint(Uri uri, CancellationToken cancellationToken);
     }
 
+    public enum IFeedUriSource
```

Comment:
* enum name shouldn't start with `I`
* Perhaps `AzDevDeploymentType` would be a clearer name?